mediator/med_phases_aofluxes_mod.F90

```diff
@@ -1109,7 +1112,8 @@ subroutine med_aofluxes_update(gcomp, aoflux_in, aoflux_out, rc)
 
     do n = 1,aoflux_in%lsize
        if (aoflux_in%mask(n) /= 0) then
-          aoflux_out%u10(n) = sqrt(aoflux_out%duu10n(n))
+          aoflux_out%u10(n)          = aoflux_out%u10res(n)
+          aoflux_out%u10_withGust(n) = sqrt(aoflux_out%duu10n(n))
```

Comment:
I find this choice of u10 != u10_withGust if using gusts confusing wrt two expectations:
-  duu10n = u10*u10 as defined above in comment as 10m wind speed (squared) 
- diagnostic u10 should be consistent with fluxes as its core physical meaning

Reply:
Have you looked through the ``cesm/flux_atmocn/shr_flux_mod.F90`` to see how these are calculated? UFS does not use the ``add_gust`` parameterizations.

Reply:
I see at least two places where diagnostic reference level wind depends on surface layer variables which change with gust:
https://github.com/NOAA-EMC/CMEPS/blob/2d837b16af326b09ff4018daab4de84f4deff7ec/cesm/flux_atmocn/shr_flux_mod.F90#L345-L400
https://github.com/NOAA-EMC/CMEPS/blob/2d837b16af326b09ff4018daab4de84f4deff7ec/cesm/flux_atmocn/shr_flux_mod.F90#L2231-L2271
although there is inevitable ambiguity when there are multiple winds. Especially if we don't use, this is probably more for consideration in ESCOMP and not this sync PR